Extension/src/LanguageServer/client.ts

```diff
@@ -650,8 +710,21 @@ class DefaultClient implements Client {
         }
     }
 
+    private prevVisibleRangesLength: number = 0;
+    private visibleRangesDecreased: boolean = false;
+    private visibleRangesDecreasedTicks: number = 0;
     public onDidChangeTextEditorVisibleRanges(textEditorVisibleRangesChangeEvent: vscode.TextEditorVisibleRangesChangeEvent): void {
         if (textEditorVisibleRangesChangeEvent.textEditor.document.uri.scheme === "file") {
+            if (vscode.window.activeTextEditor === textEditorVisibleRangesChangeEvent.textEditor) {
```

Comment:
You should probably add a comment that this tracking is just for find all references

Reply:
I added the comment above at the variable declarations.